README.md

```diff
@@ -83,6 +84,22 @@ Ergo node works normally behind NAT, so you can keep closed your `9030` port, he
 
 It is also a good practice to keep closed REST API port `9053`, and connect to your node from inside another container in the same Docker network (this case not covered by this short quick start manual).
 
+## Open project in IDE
+
+Your can use IntelliJ IDEA (Community or Ultimate edition).
```

Comment:
Maybe [Metals ](https://scalameta.org/metals/) is also woth mentioning as an alternative to Idea?